homeassistant/components/sensor/luftdaten.py

```diff
@@ -95,48 +104,53 @@ def name(self):
     @property
     def state(self):
         """Return the state of the device."""
-        return self._state
+        try:
+            return self.luftdaten.data.values[self.sensor_type]
+        except AttributeError:
```

Comment:
When could this error happen?

Reply:
Seems to be obsolete with the latest changes.